src/structures/game_session.ts

```diff
@@ -701,6 +701,18 @@ export default class GameSession {
         this.bookmarkedSongs[userID].set(song.youtubeLink, song);
     }
 
+    /** Updates owner to the first player to join the game that didn't leave VC */
+    updateOwner() {
+        const voiceMembers = getCurrentVoiceMembers(this.voiceChannelID);
+        const voiceMemberIDs = new Set(voiceMembers.map((x) => x.id));
+        const newOwnerID = ([...this.participants].filter((p) => voiceMemberIDs.has(p)))[0];
```

Comment:
What happens if the call is full of people who haven't participated yet?

Reply:
I hadn't considered that. I can check to see if that filter is of size 0, then assign it to the first person in that case